### PR TITLE
Migration to app center

### PR DESCRIPTION
**Pull request by gabrysgab**
### Task

 - [JIRA](https://netguru.atlassian.net/browse/BM-548)
 
### Description

Deleted HockeyApp references and updated readme.

### CODE REVIEW

File: app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt [hunk 1]
```diff
@@ -22,6 +21,5 @@ import javax.inject.Inject
 class DebugMetricsHelper @Inject constructor() {
 
     internal fun init(context: Context) {
```
Comment (netguru-hound) [inline app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/1cb717f0cb8fe0d6e255d90e87656ab0740ab3b4/app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt#L23"></span>Function parameter context is unused.</td>
    </tr>
  </tbody>
</table>

File: app/src/releasePreprod/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt [hunk 2]
```diff
@@ -23,7 +22,6 @@ import javax.inject.Inject
 class DebugMetricsHelper @Inject constructor() {
 
     internal fun init(context: Context) {
```
Comment (netguru-hound) [inline app/src/releasePreprod/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/1cb717f0cb8fe0d6e255d90e87656ab0740ab3b4/app/src/releasePreprod/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt#L24"></span>Function parameter context is unused.</td>
    </tr>
  </tbody>
</table>

**Comment by netguru-hound** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/netguru/baby-monitor-client-android/blob/1cb717f0cb8fe0d6e255d90e87656ab0740ab3b4/app/src/main/AndroidManifest.xml#L0">app/src/main/AndroidManifest.xml#L0</a> - The project references RTL attributes, but does not explicitly enable or disable RTL support with <code>android:supportsRtl</code> in the manifest</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/netguru/baby-monitor-client-android/blob/1cb717f0cb8fe0d6e255d90e87656ab0740ab3b4/app/src/main/AndroidManifest.xml#L20">app/src/main/AndroidManifest.xml#L20</a> - On SDK version 23 and up, your app data will be automatically backed up, and restored on app install. Your GCM regid will not work across restores, so you must ensure that it is excluded from the back-up set. Use the attribute <code>android:fullBackupContent</code> to specify an <code>@xml</code> resource which configures which files to backup. More info: https://developer.android.com/training/backup/autosyncapi.html</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/netguru/baby-monitor-client-android/blob/1cb717f0cb8fe0d6e255d90e87656ab0740ab3b4/app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt#L1">app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt#L1</a> - File must end with a newline (\n)</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:link: <a href="https://netguru.atlassian.net/browse/BM-548">BM-548</a></td>
    </tr>
  </tbody>
</table>

### JaCoCO Code Coverage 8.39% :white_check_mark:
| Class | Covered | Meta | Status |
|:---|:---:|:---:|:---:|
| `co/netguru/baby/monitor/client/feature/settings/ClientSettingsFragment` | 0% | 0% | :white_check_mark: |
| `co/netguru/baby/monitor/client/application/DebugMetricsHelper` | 0% | 0% | :white_check_mark: |


#### APK file size

| size |
| --- |
| 21.9MB |



#### Number of method references

| Dex file | count |
| --- | --- |
| classes2.dex | 22273 |
| classes.dex | 22108 |



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt (hunk 1 above)
Comment (kkogut95) [reply app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt]: @gabrysgab Do you need to pass context to this function after this change?

File: app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt [hunk 3]
```diff
@@ -1,8 +1,6 @@
 package co.netguru.baby.monitor.client.application
```
Comment (netguru-hound) [inline app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/aba9da1933f8e7bef724d75a265c8cb0a432a406/app/src/release/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt#L1"></span>File must end with a newline (\n)</td>
    </tr>
  </tbody>
</table>

**Pull request merged**